content/governing-content/identify-tasks-skills/using-presentation.md

```diff
@@ -1,11 +1,16 @@
 ---
-layout: text/textblock
-section: Content lifecycle stages
+layout: blocks/image_content_block
+image: standard/poster-stack-4up.png
```

Comment:
@meaghan-n how come the poster image is being included here?
When I made reference to the downloadable links on the /Standard page, I was only talking about the format of the hyperlink.

Reply:
I copied this across as a placeholder in case there was an image from the file that could be used in the same place. It can come out.